dev/group_vars/apim-analytics.yml

```diff
@@ -22,6 +22,12 @@ target: /mnt
 product_package_location: files
 jdbc_driver: mysql-connector-java-5.1.47-bin.jar
```

Comment:
Shall we comment these as well?

Reply:
Done https://github.com/wso2/ansible-apim/pull/42/commits/aa81e1f642134bb0e7a52351e806172549cce2f9